apps/src/music/blockly/MusicBlocklyWorkspace.js

```diff
@@ -292,6 +292,19 @@ export default class MusicBlocklyWorkspace {
     return this.workspace.getAllBlocks();
   }
 
+  updateHighlightedBlocks(playingBlockIds, notPlayingBlockIds) {
+    // Some blocks might appear in both lists, such as when a
+    // block is in a loop and only one instance is actively playing.
+    // For that reason, we clear the highlights for the not playing list
+    // first, and then highlight for the playing list second.
+    notPlayingBlockIds.forEach(blockId => {
+      Blockly.mainBlockSpace.highlightBlock(blockId, false);
+    });
```

Comment:
Do we need to keep track of the not playing list separately? Could we just clear all highlights, and then only highlight playing block IDs?

Reply:
Good call.  It was a potential optimization, and in theory could reduce flickering, but in reality, it didn't seem necessary, and the simplification is nice.